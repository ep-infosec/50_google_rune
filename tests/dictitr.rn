//  Copyright 2021 Google LLC.
//
// Licensed under the Apache License, Version 2.0 (the "License");
// you may not use this file except in compliance with the License.
// You may obtain a copy of the License at
//
//      https://www.apache.org/licenses/LICENSE-2.0
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS,
// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
// See the License for the specific language governing permissions and
// limitations under the License.

dict = Dict(string, u32)
dict.insert("Bob", 32u32)
dict.insert("Alice", 29u32)
for value in dict.values() {
  println value
}
for key in dict.keys() {
  println key
}
for pair in dict.items() {
  println (pair[0], pair[1])
}
