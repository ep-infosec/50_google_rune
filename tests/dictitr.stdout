29
32
Alice
Bob
("Alice", 29u32)
("Bob", 32u32)
